internal/prompt/prompt.go

```diff
@@ -28,6 +28,14 @@ func NewDeleteConfirm(entry string) survey.Prompt {
 	return prompt
 }
 
+// NewConfirm creates a prompt to confirm if the entry should be deleted
+func NewConfirm(message, entry string) survey.Prompt {
+	prompt := &survey.Confirm{
+		Message: fmt.Sprintf(message, entry),
+	}
+	return prompt
+}
```

Comment:
sorry if this was not clear but with the rename is to also just take the message
```suggestion
func NewConfirm(message string) survey.Prompt {
	prompt := &survey.Confirm{
		Message: message,
	}
	return prompt
}
```
and then when you call `NewConfirm` you do it like `NewConfirm(fmt.Sprintf(message, entry))`
You don't need to couple confirm and your custom format, also you could directly do 

```
&survey.Confirm{
		Message: fmt.Sprintf(message, entry),
	}
```
inside `PromptWithMessage`